tests/pfcwd/test_pfcwd_function.py

```diff
@@ -61,7 +61,7 @@ def isBufferInApplDb(asic):
             logger.info(
                 "Buffer model is {}, buffer tables will be fetched from {}".format(
                     PfcCmd.buffer_model or "not defined",
-                    "APPL_DB" if PfcCmd.buffer_model else "CONFIG_DB"
+                    "APPL_DB" if PfcCmd.buffer_model == "dynamic" else "CONFIG_DB"
```

Comment:
Just FYI. The original logic is like this:
- If `buffer_model` is defined, regardless of whether it's `dynamic` or `traditional`, it is in the `APPL_DB`. This is the image with dynamic buffer calculation supported, eg., the 202012 and newer versions.
  Even for the traditional model, all the configurations are in `CONFIG_DB`, the `buffermgr` has a logic of copying the tables from the `CONFIG_DB` to `APPL_DB` and what the `bufferorch` consumes is the entries in `APPL_DB`.
  This means in new images, the buffer configuration in `APPL_DB` takes effect eventually regardless of the buffer model. To always read buffer configuration from `APPL_DB` should be better.
- If `buffer_model` is not defined, this is the image without dynamic buffer calculation supported (probably 201911 or older). In this case, all buffer configurations are in `CONFIG_DB` and the `bufferorch` consumes the entries in `CONFIG_DB`.

Nevertheless, the change works as well and I don't have a strong opinion regardless.